### PR TITLE
Paddle payment details

### PR DESCRIPTION
**Pull request by nm**
This PR adds paddle card or paypal details to pay charges.

### CODE REVIEW

File: lib/pay/paddle/webhooks/subscription_payment_succeeded.rb [hunk 1]
```diff
@@ -32,6 +34,30 @@ def notify_user(user, charge)
...
+          when "card"
+            {
+              card_type: payment_information[:card_type],
+              card_last4: payment_information[:last_four_digits],
+              card_exp_month: payment_information[:expiry_date].split("/").first,
+              card_exp_year: payment_information[:expiry_date].split("/").last
+            }
+          when "paypal"
+            {
+              card_type: "PayPal"
```
Comment (excid3) [inline lib/pay/paddle/webhooks/subscription_payment_succeeded.rb:55]: Do they give you the PayPal email address? I've been storing the paypal email in `card_last4` for Braintree.

Comment (nm) [reply lib/pay/paddle/webhooks/subscription_payment_succeeded.rb:55]: Unfortunately not: https://developer.paddle.com/api-reference/subscription-api/users/listusers

Comment (excid3) [reply lib/pay/paddle/webhooks/subscription_payment_succeeded.rb:55]: That's too bad, but oh well.

**Comment by excid3** [comment]
Thanks for this!

**Pull request merged**